Pixel Game/Assets/Scripts/NpcData.cs

```diff
@@ -37,7 +37,8 @@ private void OnTriggerEnter2D(Collider2D collision)
             // Some npcs might not have a sound attached which is fine. Therefor we check if its null
             if(npcTalkSound != null)
             {
-                AudioSource.PlayClipAtPoint(npcTalkSound, mainCamera.transform.position);
+
+                AudioSource.PlayClipAtPoint(npcTalkSound, mainCamera.transform.localPosition);

```

Comment:
Why use `localPosition`?

Reply:
Tried with local position to counter the weird sound bug that happens when we walk around with the player after hearing a voice line.
This seemed to make the problem better (but not perfect)